### ISSUE TITLE
✨ [REQUEST] Jellyseerr

### ISSUE THREAD

**Issue by hacshacdgacs** [alexbelgium/hassio-addons#439]
**Which addon?**

Jellyseerr

**Is your feature request related to a problem? Please describe.**
Jellyseer is a branch of Overseerr that's designed to use Emby/Jellyfin instead of Plex.

**Additional context**

[[jellyseerr](https://github.com/Fallenbagel/jellyseerr)](https://hub.docker.com/r/fallenbagel/jellyseerr)

Thanks btw, unsure if you're still wanting requests to be raised as issues or in the discussions forum.

**Comment by alexbelgium** [comment]
Hi, requests are better here but I'm really busy so not sure I'll be able to take some in near future

**Comment by alexbelgium** [comment]
Hi, actually support for jellyfin and emby is coming to overseerr : https://github.com/sct/overseerr/pull/3015#issuecomment-1416707807

**Comment by mihaiaka** [comment]
Thank you for taking the time to answer back. Good catch. Will keep an eye on that.

**Comment by alexbelgium** [comment]
I've pushed the addon. Please ignore the message about the fact that there is an issue with /data/config, it is not correct. Please let me know how it goes once it has finished to build

**Comment by alexbelgium** [comment]
Thanks, indeed and I've pushed a new version that could solve it

**Comment by hacshacdgacs** [comment]
I can confirm that the same as above is happening for me on my own system, same log results also.

**Comment by mihaiaka** [comment]
Hello again,

Might need some extra help from you. When I open the web UI for jellyseerr, I'm not able to reach the jellyfish server, as per shown in the below image:
![image](https://user-images.githubusercontent.com/50486008/236778399-63ef926a-9b4a-4803-8133-4575cb4942a2.png)

For reference: 
1) everything (home assistant with add-on) runs on one raspberry pi 4, with internal IP 192.168.1.45
2) Jellyfin is reachable via port 8096, thus http://192.168.1.45:8096
3) Jellyseerr is reachable via port 5055, thus http://192.168.1.45:5055

extra check: my old jellyfin server that I'm moving away from in the near future, still runs on a totally different machine and it's reachable at http://192.168.1.200:8096/. The same situation happens with that one as well. (which makes me think it's not a network/docker/etc issue).

Also, after confirming that the server started via version 1.5.0-6 last week, I did not had the time to check it further, so I cannot relate for sure if this situation occurred in that one as well, but in the meantime is see that the add-on updated to "338" (anyway I could roll back to that one to check?)

Thank you.

**Comment by alexbelgium** [comment]
Hi, strange as I could connect to my emby server straight away. Are you sure the setting in the options is set to Jellyfin? And that you jellyfin port is still on the default 8096 (probably doesn't matter as it connects to the host network). Any element on the jellyfin log (either addon or app) ?

**Comment by mihaiaka** [comment]
Hi,

In the above image, you can see on the far right side the browser being able to open the web UI of Jellyfin, so this should confirm the (default) port still being 8096.
The confirmation tab for jellyseerr looks ok:
![image](https://user-images.githubusercontent.com/50486008/236785260-e4edddf5-cfd9-4bdf-8783-b638ec11263c.png)

**Log for jellyseer:**
Starting...
/etc/cont-init.d/00-banner.sh: executing
-----------------------------------------------------------
 Add-on: Jellyseerr
 fork of overseerr for jellyfin support
-----------------------------------------------------------
 Add-on version: 338
 You are running the latest version of this add-on.
 System: Home Assistant OS 10.1  (aarch64 / raspberrypi4-64)
 Home Assistant Core: 2023.5.2
 Home Assistant Supervisor: 2023.04.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums
-----------------------------------------------------------
 Provided by: https://github.com/alexbelgium/hassio-addons 
-----------------------------------------------------------
/etc/cont-init.d/00-local_mounts.sh: executing
/etc/cont-init.d/00-smb_mounts.sh: executing
/etc/cont-init.d/01-custom_script.sh: executing
[11:07:01] INFO: Execute /config/addons_autoscripts/jellyseerr.sh if existing
[11:07:01] INFO: ... no script found
/etc/cont-init.d/90-run.sh: executing
[11:07:02] INFO: Config stored in /config/addons_config/jellyseer
yarn run v1.22.19
$ NODE_ENV=production node dist/index.js
2023-05-08T09:07:29.424Z [info]: Commit Tag: $GIT_SHA 
2023-05-08T09:07:31.456Z [info]: Starting Overseerr version 1.5.0 
warn  - You have enabled experimental features (scrollRestoration, largePageDataBytes) in next.config.js.
warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.
2023-05-08T09:07:37.079Z [info][Notifications]: Registered notification agents 
2023-05-08T09:07:37.241Z [info][Jobs]: Scheduled jobs loaded 
2023-05-08T09:07:38.375Z [info][Server]: Server ready on port 5055 
2023-05-08T09:08:00.047Z [debug][Jobs]: Starting scheduled job: Download Sync 
2023-05-08T09:09:00.029Z [debug][Jobs]: Starting scheduled job: Download Sync 
2023-05-08T09:10:00.026Z [debug][Jobs]: Starting scheduled job: Download Sync 
2023-05-08T09:10:00.074Z [info][Jobs]: Starting scheduled job: Plex Watchlist Sync 
2023-05-08T09:10:00.126Z [info][Jobs]: Starting scheduled job: Jellyfin Recently Added Sync 
2023-05-08T09:10:00.139Z [info][Jellyfin Sync]: Jellyfin Sync Starting {"sessionId":"dd78c357-dfb6-4d02-aaeb-629623ee544a"}
2023-05-08T09:10:00.191Z [info][Jellyfin Sync]: Recently Added Scan Complete 
2023-05-08T09:11:00.034Z [debug][Jobs]: Starting scheduled job: Download Sync 

**Logs for jellyfin:**
cont-init: info: running /etc/cont-init.d/00-local_mounts.sh
cont-init: info: /etc/cont-init.d/00-local_mounts.sh exited 0
cont-init: info: running /etc/cont-init.d/00-smb_mounts.sh
cont-init: info: /etc/cont-init.d/00-smb_mounts.sh exited 0
cont-init: info: running /etc/cont-init.d/01-custom_script.sh
[11:17:00] INFO: [32mExecute /config/addons_autoscripts/jellyfin.sh if existing[0m
[11:17:00] INFO: [32m... no script found[0m
cont-init: info: /etc/cont-init.d/01-custom_script.sh exited 0
cont-init: info: running /etc/cont-init.d/20-folders.sh
Creating /jellyfin
Creating link for /jellyfin/cache
Creating link for /jellyfin/data
Creating link for /jellyfin/log
Creating link for /jellyfin/metadata
Creating link for /jellyfin/plugins
Creating link for /jellyfin/root
cont-init: info: /etc/cont-init.d/20-folders.sh exited 0
cont-init: info: running /etc/cont-init.d/32-ingress.sh
cont-init: info: /etc/cont-init.d/32-ingress.sh exited 0
cont-init: info: running /etc/cont-init.d/90-data_location.sh
[11:17:01] INFO: [32mSetting data location to /config/addons_config/jellyfin[0m
Creating /config/addons_config/jellyfin
cp: '/config/addons_config/jellyfin/data/transcodes' and '/data/transcodes' are the same file
cont-init: info: /etc/cont-init.d/90-data_location.sh exited 1
cont-init: info: running /etc/cont-init.d/99-database_clean.sh
cont-init: info: /etc/cont-init.d/99-database_clean.sh exited 0
[custom-init] No custom files found, skipping...
[11:17:05] [INF] [1] Main: Jellyfin version: 10.8.10
[11:17:05] [INF] [1] Main: Environment Variables: ["[JELLYFIN_DATA_DIR, /config/addons_config/jellyfin/data]", "[JELLYFIN_WEB_DIR, /config/addons_config/jellyfin/web]", "[JELLYFIN_CONFIG_DIR, /config/addons_config/jellyfin]", "[JELLYFIN_CACHE_DIR, /config/addons_config/jellyfin/cache]", "[JELLYFIN_LOG_DIR, /config/addons_config/jellyfin/log]"]
[11:17:05] [INF] [1] Main: Arguments: ["/usr/lib/jellyfin/bin/jellyfin.dll", "--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg"]
[11:17:05] [INF] [1] Main: Operating system: Linux
[11:17:05] [INF] [1] Main: Architecture: Arm64
[11:17:05] [INF] [1] Main: 64-Bit Process: True
[11:17:05] [INF] [1] Main: User Interactive: True
[11:17:05] [INF] [1] Main: Processor count: 4
[11:17:05] [INF] [1] Main: Program data path: /config/addons_config/jellyfin/data
[11:17:05] [INF] [1] Main: Web resources path: /config/addons_config/jellyfin/web
[11:17:05] [INF] [1] Main: Application directory: /usr/lib/jellyfin/bin/
[11:17:06] [INF] [1] Emby.Server.Implementations.AppBase.BaseConfigurationManager: Setting cache path: /config/addons_config/jellyfin/cache
[11:17:06] [INF] [1] Emby.Server.Implementations.ApplicationHost: Loading assemblies
[11:17:06] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN addresses : [10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]
[11:17:06] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Defined LAN exclusions : []
[11:17:06] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using LAN addresses: [10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]
[11:17:06] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind addresses: []
[11:17:06] [INF] [1] Jellyfin.Networking.Manager.NetworkManager: Using bind exclusions: []
[11:17:17] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: TMDb 10.8.10.0
[11:17:17] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: Studio Images 10.8.10.0
[11:17:17] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: OMDb 10.8.10.0
[11:17:17] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: MusicBrainz 10.8.10.0
[11:17:17] [INF] [1] Emby.Server.Implementations.Plugins.PluginManager: Loaded plugin: AudioDB 10.8.10.0
[11:17:17] [INF] [1] Main: Kestrel listening on Any IP4 Address
[11:17:19] [WRN] [1] Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager: No XML encryptor configured. Key {b60fd54e-92b8-443a-8b7b-efe9a2084761} may be persisted to storage in unencrypted form.
[11:17:19] [INF] [1] Emby.Server.Implementations.ApplicationHost: Running startup tasks
services-up: info: copying legacy longrun nginx (no readiness notification)
[ls.io-init] done.
[11:17:19] [INF] [1] Emby.Server.Implementations.ScheduledTasks.TaskManager: Daily trigger for Extract Chapter Images set to fire at 2023-05-09 02:00:00.000 +02:00, which is 14:42:40.0821875 from now.
[11:17:20] INFO: [32mStarting NGinx...[0m
[11:17:20] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Found ffmpeg version 5.1.3
[11:17:20] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available decoders: ["libdav1d", "av1", "h264", "hevc", "mpeg2video", "mpeg4", "msmpeg4", "vp8", "libvpx", "vp9", "libvpx-vp9", "aac", "ac3", "dca", "flac", "mp3", "truehd"]
[11:17:20] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available encoders: ["libx264", "h264_v4l2m2m", "libx265", "mpeg4", "msmpeg4", "libvpx", "libvpx-vp9", "aac", "libfdk_aac", "ac3", "dca", "flac", "libmp3lame", "libopus", "truehd", "libvorbis", "srt"]
[11:17:20] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available filters: ["hwupload_vaapi", "zscale", "alphasrc"]
[11:17:20] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: scale_cuda with option Output format (default "same") is not available
[11:17:21] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: tonemap_cuda with option GPU accelerated HDR to SDR tonemapping is not available
[11:17:21] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: tonemap_opencl with option bt2390 is not available
[11:17:21] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: overlay_opencl with option Action to take when encountering EOF from secondary input is not available
[11:17:21] [WRN] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Filter: overlay_vaapi with option Action to take when encountering EOF from secondary input is not available
[11:17:21] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Available hwaccel types: ["drm"]
[11:17:21] [WRN] [11] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://192.168.1.45:8096/ScheduledTasks?IsEnabled=true to 192.168.1.134 in 0:00:01.569444 with Status Code 503
[11:17:21] [WRN] [8] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://192.168.1.45:8096/system/info/public to 192.168.1.134 in 0:00:01.5680546 with Status Code 503
[11:17:21] [WRN] [10] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://192.168.1.254:8096/system/info/public to 192.168.1.134 in 0:00:01.5735531 with Status Code 503
[11:17:21] [WRN] [16] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/health to 127.0.0.1 in 0:00:00.9350964 with Status Code 503
[11:17:21] [INF] [1] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: FFmpeg: /usr/lib/jellyfin-ffmpeg/ffmpeg
[11:17:21] [INF] [1] Emby.Server.Implementations.ApplicationHost: ServerId: 1512c5f527d14f66bc8e7cb34e0181bc
[11:17:22] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all pre-startup entry points in 0:00:00.4809616
[11:17:22] [INF] [1] Emby.Server.Implementations.ApplicationHost: Core startup complete
[11:17:22] [INF] [11] Emby.Server.Implementations.ScheduledTasks.TaskManager: StartupTrigger fired for task: Update Plugins
[11:17:22] [INF] [11] Emby.Server.Implementations.ScheduledTasks.TaskManager: Queuing task PluginUpdateTask
[11:17:23] [INF] [8] Emby.Server.Implementations.ScheduledTasks.TaskManager: Executing Update Plugins
[11:17:23] [WRN] [10] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://192.168.1.254:8096/ScheduledTasks?IsEnabled=true to 192.168.1.134 in 0:00:00.6589871 with Status Code 200
[11:17:23] [INF] [16] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /share/downloads/Movies
[11:17:23] [INF] [11] Emby.Server.Implementations.IO.LibraryMonitor: Watching directory /share/downloads/TV
[11:17:23] [INF] [1] Emby.Server.Implementations.ApplicationHost: Executed all post-startup entry points in 0:00:01.4586034
[11:17:23] [INF] [1] Main: Startup complete 0:00:19.7394072
[11:17:25] [INF] [16] Emby.Server.Implementations.ScheduledTasks.TaskManager: Update Plugins Completed after 0 minute(s) and 2 seconds
[11:17:25] [INF] [16] Emby.Server.Implementations.ScheduledTasks.TaskManager: ExecuteQueuedTasks
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'AccessSchedule' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ActivityLog' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'CustomItemDisplayPreferences' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'DisplayPreferences' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'HomeSection' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ImageInfo' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ItemDisplayPreferences' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'Permission' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'Preference' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'ApiKey' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'Device' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'DeviceOptions' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Microsoft.EntityFrameworkCore.Model.Validation: The entity type 'User' is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider.
[11:17:27] [WRN] [16] Jellyfin.Server.Middleware.ResponseTimeMiddleware: Slow HTTP Response from http://127.0.0.1:8096/health to 127.0.0.1 in 0:00:00.5691539 with Status Code 200
[11:17:28] [INF] [8] Emby.Dlna.Main.DlnaEntryPoint: DLNA Session created for [TV]UE46ES6100 - UE46ES6100

**Comment by alexbelgium** [comment]
Mmh this is quite surprising. I've stopped emby (to make sure nothing else runs on port 8096), installed Jellyfin, cleaned jellyseer data (to remove my installation with emby), then restarted. I encoded my data as shown in the image, then it worked straight away... Are you sure you don't have something such as a subpath, or whitelist, that might impact ? As on my end it works both with emby and jellyfin

<img width="533" alt="je" src="https://user-images.githubusercontent.com/44178713/236813938-c3799eab-ceff-4002-8985-440b46fa6692.png">

**Comment by mihaiaka** [comment]
Not sure what's wrong either :(
"subpath, or whitelist", I did not make any changes (that would complicate things).
"cleaned jellyseer data" Maybe I should try that? Is it safe if I empty this folder after an uninstall? followed up a fresh install? /config/addons_config/jellyseer

The fact that Jellyfin uses the "host" network in docker and jellyseerr uses the "hassio" should not be any problem? right? they would eventually meet in the host part of 192.168.1.0/24

**Comment by alexbelgium** [comment]
Searching on google for "something went wrong while trying to sign in" and overseerr or jellyseerr doesn't seem to bring really meaningful elements

**Comment by alexbelgium** [comment]
It's funny there is really no error message at all in overseerr log... And I've seen your message about impact of version change, but actually it is linked to the latest tag last updated 18 days ago so it is actually the same

**Comment by mihaiaka** [comment]
yes, it's HAos, on a RPI 4 

the only place I saw this ever mentioned, was a UNRAID forum:
https://forums.unraid.net/topic/125062-jellyseerr-jam-packed-juggernaut-media-requests-overseerr-fork/?do=findComment&comment=1220839
but nobody replied to his inquiry.

I went along with uninstalling 338, deleted /config/addons_config/jellyseer and went back to reinstall. 338-2 was already uploaded, so I went and installed that one.

and...

![image](https://user-images.githubusercontent.com/50486008/236863394-2153e624-0e96-4cd1-b7c4-a7c094bde4a5.png)

It works! :)
Unfortunately... as I did both the deletion of the config and installed your updated version ("forced it on the host network") I cannot pun my finger on which was the actual resolution :) sorry about that.

![image](https://user-images.githubusercontent.com/50486008/236864887-739a6782-9c30-453c-9c59-561af8e96dca.png)


Thank you very much for your time.
